### PR TITLE
check if intro skip button is build-in and skip injecting

### PR DESCRIPTION
**Pull request by jumoog**

### CODE REVIEW

**Comment by jumoog** [comment]
ok my check works:
`2024-08-27T18:30:14.145883055Z [20:30:14] [INF] [1] ConfusedPolarBear.Plugin.IntroSkipper.Plugin: jellyfin has build-in skip button`

File: .github/workflows/webui.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        node-version: [20.x]
+        jellyfin-web-version: [10.9.10]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
```
Comment (jumoog) [inline .github/workflows/webui.yml:21]: newest version is actions/setup-node@v4

**Comment by jumoog** [comment]
ready?

**Comment by rlauuzo** [comment]
Ja.

**Pull request merged**